requirements.txt

```diff
@@ -22,7 +22,7 @@ pika == 1.2.1
 brainzutils@git+https://github.com/metabrainz/brainzutils-python.git@upgrade-deps
 spotipy>=2.22.1
 datasethoster@git+https://github.com/metabrainz/data-set-hoster.git@a1e12968af93d2f296a42a8094ebff83f3ed33f3
-troi@git+https://github.com/metabrainz/troi-recommendation-playground.git@v-2023-10-30.0
+troi@git+https://github.com/metabrainz/troi-recommendation-playground.git@integration-fixes
```

Comment:
Best to replace with exact commit hash to ensure latest commit is used.

Reply:
Yes, once the troi PR is merged, I will make a release and then update this string, before merging.